pyjokes/jokes_de.py

```diff
@@ -73,13 +66,11 @@
     u"Chuck Norris hat einen Grizzlybär-Teppich. Der Bär lebt, hat bloß Angst, sich zu bewegen.",
     "Wenn Chuck Norris mit nassen Fingern an der Steckdose rumfummelt, kriegt das Kraftwerk einen Schlag.",
     "Chuck Norris kann mit Durchfall furzen.",
-    "Chuck Norris klaut den Polen die Autos.",
     u"Wenn Chuck Norris einen Raum betritt, stehen sogar die Stühle auf!",
     "Chuck Norris hat bei Burgerking einen Big Mac bestellt - und hat ihn ohne Wiederworte bekommen",
     u"Chuck Norris war bereits auf dem Mars. Das ist der Grund dafür, warum es dort kein Leben mehr gibt.",
     "Chuck Norris wurde letztens von der Polizei angehalten... Die Polizisten sind mit einer Verwarnung davon gekommen.",
     "Chuck Norris kackt Lichtschwerter.",
-    u"Herr Norris benutzt keine Kondome, denn es gibt nichts, was vor ihm schützen könnte.",
```

Comment:
I had my issues with this. A joke about condoms also imply that it is necessary to use condoms.